src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
```

Comment:
The `getRandomBall()` method is currently static. According to the checklist, static methods are generally considered bad practice. It's better to create an instance of the `Lottery` class and call the method on that instance .